src/nuance_assets/components/search-results/search-results.tsx

```diff
@@ -294,19 +294,40 @@ export const SearchResults = (props: {
             />
           </div>
           <Dropdown
+            className='desktop-only-flex'
             uniqueId={'search-results-dropdown-menu'}
+            style={{
+              alignSelf: 'end',
+              width: '120px',
+              height: '27px',
+              top: '0',
+              position: 'absolute',
+              right: '0',
+            }}
+            selectedTextStyle={{ fontSize: '14px', fontWeight: '400' }}
             items={['Relevance', 'Most recent']}
             onSelect={(item: string) => {
               setSortType(item as SortType);
             }}
+            drodownItemsWrapperStyle={{
+              top: '28px',
+              rowGap: 0,
+              minWidth: '120px',
+            }}
+            arrowWidth={12}
+          />
+          <Dropdown
+            className='mobile-only-flex'
+            uniqueId={'search-results-dropdown-menu-mobile'}
             style={{
-              width: '100px',
-              height: '25px',
-              position: 'absolute',
-              right: '0',
-              bottom: '0',
+              width: '120px',
+              height: '27px',
+            }}
+            selectedTextStyle={{ fontSize: '14px', fontWeight: '400' }}
+            items={['Relevance', 'Most recent']}
```

Comment:
It seems from the design, there should be a "Sort" label on the menu. Relevance/Most recent should be in the drop down menu. I'm not seeing much else different, maybe the color is slightly off? The design shows the active state as #1BC1F0